server.py

```diff
@@ -64,7 +60,8 @@ def get_version():
 def post_collect():
     """Endpoint servicing data collection."""
     input_data = request.get_json(force=True)
-    validation = SCHEMA.load(input_data)
```

Comment:
is this validation only applicable to the clustering worker? if so, let's move to its own scope.
Leaving this block is quite confusing to other contexts.

Reply:
This validation is valid for any context, since we're expecting the automated trigger (incoming listener) to forward us the message. However the content is optional anyways, we're not enforcing presence of any data. Except it has to be a JSON payload (which is actually enforced one line above). We're enforcing a payload because its a POST request (requesting us to queue a job)... I think it's fine here.

Reply:
For either case of 
1) empty payload
2) non-json payload
the line before this validation block: `input_data = request.get_json(force=True)` is already failing the request with `400`. So if this validation block (line 67-77) only serves to detect against the above 2 cases, it is redundant.

If individual collector has more sophisticated validation logic, they can expose a `validation` method that we can invoke here.

Reply:
anyways, this validation and API is all just for pre-Summit. It will change with rearch, I agree, we can remove the validation and default to empty payload for now. Sounds good?

Reply:
yeah, I point it out considering this is a good fit for this (mainly) refactoring PR. 
But if you see removing this now is risky, we can wait.

Reply:
Nope, I think it make sense to remove it. The explanation you've provided is good. I've change it now, so the server can accept both types of requests - with and without payload. No scheme is enforced. On the other hand, I've made the `b64_identity` mandatory. How does that sound to you, @jameswnl ? :wink: